### PR TITLE
Remove platform requirements from images

### PR DESCRIPTION
**Pull request by h-bragg**
By having the `--ignore-platform-reqs` entry for every request, versions which are not compatible at all are attempted to be installed, case in point, the recent change by `doctrine` to have `"php": "^7.1"` as one of their dependencies.

If you need to target a specific version of php or an extension, can can specify a mock `platform` in the `composer.json` file:

```json
{
  "config": {
    "platform": {
      "php": "5.6",
      "ext-mbstring": "1.2.1"
    }
  }
}
```

### CODE REVIEW

File: tests/graze_composer_env.bats [hunk 1]
```diff
@@ -51,136 +51,130 @@ teardown() {
...
-@test "the image entrypoint should be the composer wrapper" {
+@test "the image entrypoint should be composer" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Entrypoint[]'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo 'status:' "$status"
+  echo 'output:' "$output"
   [ "$status" -eq 0 ]
-  [ "$output" = "/usr/local/bin/composer-wrapper" ]
+  [ "$output" = "/usr/bin/php/usr/local/bin/composer" ]
```
Comment (biggianteye) [inline tests/graze_composer_env.bats]: This looks weird. I know it's a consequence of the fact that two lines are returned from `docker inspect` as we have talked about, but it looks odd to see the values munged together like this.

Comment (h-bragg) [reply tests/graze_composer_env.bats]: this looks like an old version of the file

**Review by biggianteye — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -42,22 +42,22 @@ Combination images are in the format: `{composer_ver}-php{php_ver}`
     -v $(pwd):/usr/src/app \
     -v ~/.composer:/home/composer/.composer \
     -v ~/.ssh/id_rsa:/home/composer/.ssh/id_rsa:ro \
-   graze/composer:php-7.1
+   graze/composer
 ```
 
-### Simulating php and extension versions within composer
+### Simulatting the platform
```
Comment (biggianteye) [inline README.md]: Simulating

Comment (h-bragg) [reply README.md]: Simulating ✔️

**Comment by h-bragg** [comment]
💥

**Pull request merged**